### ISSUE TITLE
"Can't unpack Ramdisk"

### ISSUE THREAD

**Issue by Leonardo64** [ClusterM/hakchi2#1357]
This is the error the hakchi2 gives me after I try to install the Kernel. I didn't backup the Kernel. The SNES Mini is American. I'd like to know how to solve an flash the system. I seem to miss the "Dump" folder. Thanks.

**Comment by XxMcW1LL14MxX** [comment]
I Have the same problem with my SNES Classic. I accidentally downloaded a game that wasn't meant to be downloaded, then I turned on the TV and the screen was black. Then I went back and unchecked the game, then tried again on the TV and the screen was still black.

 When I tried to boot the kernel from the RAM, the thing said that there was no kernel dump and that I had to dump the kernel. When I tried to find the kernel, there was no trace of it. When I try to flash the kernel, it says "Can't unpack ramdisk".

I don't think I bricked it because I can still sync it with the computer just fine but I don't have a clue on how to fix it.

**Comment by SirBeethoven** [comment]
I need help with this too

**Comment by ClusterM** [comment]
Have you tried new version?

**Comment by Marktrip** [comment]
I have the same issue, can’t unpack ramdisk... and i have no folder dump... and i Can only get 2 games.. i’m not Einstein in this computer World, so im about to give up

**Comment by XxMcW1LL14MxX** [comment]
Yep, it is a pain in the butt. I only said "but' because I don't know
GitHub's policy on profanity...it was probably in the Terms and Conditions.
Anyway, the only good thing about this problem is that it was my fault, and
not a flawed service or whatever. I did not bother to get the latest
version because I thought there wouldn't be any use since the SNES doesn't
even turn on. Well, it turns on, but only by showing the lights on the
front. Sad day in America...or wherever you people are from.
